.editorconfig

```diff
@@ -0,0 +1,146 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
```

Comment:
Is this file added by intention?

Reply:
yes - it adds:

1. forces tab sizes for the project to 4, since VS sometimes can't auto detect it (and it seems to be the standard across the project)
2. it adds explicit ignores for `dotnet_diagnostic.CA1416.severity = silent`

VS adds all the other boilerplate junk :P I can try to cut out all the other defaults if that's preferable. Note that it shouldn't change anything significant other than the editing experience.

Reply:
Yeah it makes sense to have a uniform editor preference for everybody working on Flow. @jjw24 what do you think about this?

Reply:
Yes absolutely, I think this is a great idea and something we should have implemented a long time ago. Thank you @nachmore